src/components/app/app.scss

```diff
@@ -13,10 +13,6 @@
   .kui-theme--dark & {
     fill: $color-light;
   }
-
-  .pipeline-icon-toolbar__button:hover & {
-    opacity: 1;
-  }
```

Comment:
Why this change? Will this not totally remove all hover styling? If you want to avoid having hover states on disabled buttons, I think this would be a better approach:
```scss
.pipeline-icon-toolbar__button:not(:disabled):hover & {
  opacity: 1;
}
```

Reply:
ah I see, it's added elsewhere. Fair enough. I'm not stoked about the styles being added in two different places, but maybe we should address that in a separate PR. Nevermind :)

Reply:
I originally wrote the same thing, but then realised that there was no longer a need for that styling to be inside of `app.css` anymore and moved it to the icon toolbar styles where it would normally be.

There's no duplication, I'm not sure what you mean?

Reply:
you're right, no duplication, more that the intricately bound styles are being handled in two places. Made me think that it could be done in a more obvious manner. But it's not related to your PR so ignore for now

Reply:
I agree looks like it should be elsewhere too, I imagine the move to CSS variables will likely avoid what ever reason it was put here originally when it was first implemented.